docs/changes.rst

```diff
@@ -175,6 +175,9 @@ Other changes:
 - make RPC protocol more extensible: use named parameters.
 - RemoteRepository: misc. code cleanups / refactors
 - clarify cache/repository README file
+- the --keep-tag-files option has been deprecated in favor of --keep-exclude-tags.
+  additionally, both options now also accept directories as tags for backup
+  exclusion, #1999
```

Comment:
also, "both options now also accept directories as tags" is a bit misleading in the context of "--keep-tag-files option" and "--keep-exclude-tags" - these options do not get any NAME as value, but are just booleans.

so, this refers to the --exclude-if-present NAME options rather.